<<<<<<< HEAD
from numpy.lib import NumpyVersion

version: NumpyVersion

NumpyVersion(b"1.8.0")  # E: incompatible type
version >= b"1.8.0"  # E: Unsupported operand types
=======
from numpy.lib import NumpyVersion

version: NumpyVersion

NumpyVersion(b"1.8.0")  # E: incompatible type
version >= b"1.8.0"  # E: Unsupported operand types
>>>>>>> 4f10d78437bf31c0feec612c4720144541ce9436
