<<<<<<< HEAD
import numpy.lib.array_utils as array_utils

array_utils.byte_bounds(1)  # E: incompatible type
=======
import numpy.lib.array_utils as array_utils

array_utils.byte_bounds(1)  # E: incompatible type
>>>>>>> 4f10d78437bf31c0feec612c4720144541ce9436
