<<<<<<< HEAD
import numpy as np

np.isdtype(1, np.int64)  # E: incompatible type

np.issubdtype(1, np.int64)  # E: incompatible type
=======
import numpy as np

np.isdtype(1, np.int64)  # E: incompatible type

np.issubdtype(1, np.int64)  # E: incompatible type
>>>>>>> 4f10d78437bf31c0feec612c4720144541ce9436
